com.civicrm.hrjobroles/js/src/job-roles/controllers/hr-job-roles-controller.js

```diff
@@ -398,6 +432,8 @@ define([
 
                         // Hide the empty message if visible
                         $scope.empty = false;
+
```

Comment:
Shouldn't you keep the `// Get job roles based on the passed Contact ID (refresh part of the page)` comment?